src/test_data/arpes.py

```diff
@@ -363,7 +363,24 @@ def perpendicular_momentum(photon_energy, parallel_momentum,
     $\k_{perpendicular}$ are the surface parallel and perpendicular momentum,
     $E_{ph}$ is the photon energy, $E_{b}$ is the electron binding energy and
     $\Phi$ is the work function.
+    Parameters
+    ----------
+    photon_energy : float or 1D numpy.ndarray,
+        The photon energy (or energies) in eV.
+    parallel_momentum : float or 1D numpy.ndarray,
+        The parallel momentum in the unit of the inverse of angstrom.
+    binding_energy : float or 1D numpy.ndarray,
+        The binding energy (or energies) in eV.
+
+    inner_potential: float or 1D numpy.ndarray
```

Comment:
These two values can only be floats or integers (it makes no sense for them to vary by point) can you change this?